etc/profile-m-z/telegram.profile

```diff
@@ -36,10 +36,15 @@ noroot
 notv
 protocol unix,inet,inet6,netlink
 seccomp
+seccomp.block-secondary
 shell none
+tracelog
 
 disable-mnt
+#private-bin telegram-desktop,telegram,Telegram,telegram-desktop
```

Comment:
```suggestion
#private-bin telegram,Telegram,telegram-desktop
```

Reply:
wow... what happens to my eyes! my eyes!